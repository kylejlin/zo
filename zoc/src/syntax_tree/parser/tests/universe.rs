use super::*;

#[test]
fn set0() {
    let src = r#"Set0"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn set1() {
    let src = r#"Set1"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn set42() {
    let src = r#"Set42"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn prop0() {
    let src = r#"Prop0"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn prop1() {
    let src = r#"Prop1"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}

#[test]
fn prop42() {
    let src = r#"Prop42"#;
    let tokens = lex(src).unwrap();
    let cst = parse(tokens).unwrap();
    insta::assert_debug_snapshot!(&cst);
}
