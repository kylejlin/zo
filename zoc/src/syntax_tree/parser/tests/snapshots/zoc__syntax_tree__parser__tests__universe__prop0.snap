---
source: zoc/src/syntax_tree/parser/tests/universe.rs
expression: "&ost"
---
Universe(
    UniverseLiteral {
        level: 0,
        span: (
            ByteIndex(
                0,
            ),
            ByteIndex(
                5,
            ),
        ),
        erasable: true,
    },
)
